lang/c++/README

```diff
@@ -39,8 +39,8 @@ mkdir build
 cd build
 cmake -G "Unix Makefiles" ..
```

Comment:
Should this be updated after https://github.com/apache/avro/pull/2745 ?

Reply:
Not necessarily. This command will still work. But if I remove `-G "Unix Makefiles"` here, it might configure with `Ninja` build system. And this will make the `make` invocation` on the next line fail.
I could, of course, replace it with `cmake --build build` instead, but it seems somewhat more verbose than a simple `make`.

If needed, I can update it, of course :)